src/fu-util.c

```diff
@@ -2736,6 +2876,12 @@ main (int argc, char *argv[])
 		     /* TRANSLATORS: command description */
 		     _("Reinstall current firmware on the device."),
 		     fu_util_reinstall);
+	fu_util_cmd_array_add (cmd_array,
+		     "switch-branch",
```

Comment:
aww you didn't like my command name `sidegrade`?  I thought it captured the possibility for firmware version schema being entirely different.

Reply:
I didn't like `sidegrade` for three reasons:

 * It didn't include the word branch, and that's what we've used everywhere else
 * It's not sideloading a firmware from another repo like you would on a phone
 * It's not really understandable if your primary language isnt' `en_*`